index.html

```diff
@@ -47,142 +53,157 @@ <h2>Software Developer</h2>
                 </div>
-                <button class="button1" type="button">See Project</button>
-            </div>
-            <div class="work-portofolio-3">
-                <div class="grid-container">
-                    <div class="grid-item1"></div>
-                    <div class="grid-item2">
-                        <h2>Multi-Post Stories</h2>
-                    </div>
-                    <div class="grid-item3">
-                        <p>
-                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                        </p>
-                    </div>
-                    <div class="grid-item4">
-                        <ul>
-                            <li>css</li>
-                            <li>html</li>
-                            <li>bootstrap</li>
-                            <li>Ruby</li>
-                        </ul>
-                    </div>
+                <div class="grid-item4">
+                    <ul>
+                        <li>css</li>
+                        <li>html</li>
+                        <li>bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
                 </div>
-                <button class="button1" type="button">See Project</button>
-            </div>
-            <div class="work-portofolio-4">
-                <div class="grid-container">
-                    <div class="grid-item1"></div>
-                    <div class="grid-item2">
-                        <h2>Multi-Post Stories</h2>
-                    </div>
-                    <div class="grid-item3">
-                        <p>
-                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                        </p>
-                    </div>
-                    <div class="grid-item4">
-                        <ul>
-                            <li>css</li>
-                            <li>html</li>
-                            <li>bootstrap</li>
-                            <li>Ruby</li>
-                        </ul>
-                    </div>
+                <div class="grid-item5">
+                    <button class="buttonGR" type="button">See Project</button>
                 </div>
-                <button class="button1" type="button">See Project</button>
             </div>
+            <button class="button1" type="button">See Project</button>
         </div>
     </section>
     <section>
         <div class="Contact-Us">
             <div class="contact-title">
-                <h2>CONTACT US</h2>
+                <h2 class="contact-title-sm">CONTACT US</h2>
+                <h2 class="contact-title-LG">Let’s chat!</h2>
```

Comment:
- [ ] your contact section and footer looks really small. Kindly follow the font size and font weight of the figma desktop version